libs/prelude/Prelude/Ops.idr

```diff
@@ -11,6 +11,8 @@ infixr 4 ||
 
 -- List and String operators
 infixr 7 ::, ++
+infixl 7 :<, <><
+infixr 6 <>>
```

Comment:
Isn't it a bit strange to declare fish & ships in `prelude` even though they're only defined in `base`?